client/data/promotions/actions.ts

```diff
@@ -0,0 +1,123 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { apiFetch } from '@wordpress/data-controls';
+import { controls } from '@wordpress/data';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ACTION_TYPES from './action-types';
+import {
```

Comment:
```suggestion
import type {
```

A small readability improvement: either prepend the full import path once or prepend it on each line.

The same adjustment can be made in the other files as well.

Reply:
Great suggestion, thanks! Is [this](https://github.com/Automattic/woocommerce-payments/pull/11131/commits/96d8d64112c7becb477e874c9500ae4f54c782d5) what you had in mind?

Reply:
Yes! Thank you for improving it!